test/topology_custom/test_tablets.py

```diff
@@ -324,3 +325,44 @@ async def test_read_of_pending_replica_during_migration(manager: ManagerClient,
 
     rows = await cql.run_async("SELECT pk from test.test")
     assert len(list(rows)) == 1
+
+
+# This test checks that --enable-tablets option and the TABLETS parameters of the CQL CREATE KEYSPACE
+# statemement are mutually correct from the "the least surprising behavior" concept. See comments inside
+# the test code for more details.
+@pytest.mark.parametrize("with_tablets", [True, False])
+@pytest.mark.parametrize("replication_strategy", ["NetworkTopologyStrategy", "SimpleStrategy", "EverywhereStrategy", "LocalStrategy" ])
+@pytest.mark.asyncio
+async def test_keyspace_creation(manager: ManagerClient, with_tablets, replication_strategy):
+    cfg = {'enable_user_defined_functions': False, 'enable_tablets': with_tablets}
```

Comment:
Is `'enable_user_defined_functions': False` required?

Reply:
For tablets to be enabled. Other tests have it

Reply:
Hm, it seems that this is no longer a requirement. Will remove